src/api/routes/myGroups.ts

```diff
@@ -23,56 +25,64 @@ const myGroups = router({
    * TODO: Only allow group users to call this function.
    */
   generateMeetingLink: procedure
-  .use(authUser())
```

Comment:
Can you undo these large-scale auto formatting changes? It makes review very difficult.

We can check in a PR dedicated to format changes.

Also, please self-review a PR before submitting for review. Thanks~